testing/xfpga/test_enum_c.cpp

```diff
@@ -74,6 +74,58 @@ class enum_c_p : public ::testing::TestWithParam<std::string> {
     }
   }
 
+  // Need a concrete way to determine the number of fpgas on the system
+  // without relying on fpgaEnumerate() since that is the function that
+  // is under test. 
+  int GetNumFpgas() {
+    if (platform_.mock_sysfs != nullptr) {
+      return platform_.devices.size();
+    }
+
+    int value;
+    std::string cmd = "lspci | "
```

Comment:
grep 'intel-fpga-dev' /sys/class/fpga | wc -l
Would also work, and it would be closer to the driver.